.github/workflows/integration-test-multi-cluster-suite.yaml

```
@@ -18,6 +18,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
   cancel-in-progress: true
 
+  permissions:
+    contents: read
```

I'm pretty sure the top-level permissions config needs to be left aligned. As written, I think this interprets as being part of the concurrency section above.

```suggestion
permissions:
  contents: read
```

---

I still see a handful of places where this is indented at the top level.

---

@BlaineEXE  wrote:

> I still see a handful of places where this is indented at the top level.

I think now I have found and fixed all cases